build-conf/README.md

```diff
@@ -98,6 +99,7 @@ bms_loadOptions | BPXWDYN creation options for 'load module' type data sets
 bms_tempOptions | BPXWDYN creation options for temporary data sets
 bms_assembler | MVS program name of the high level assembler
 bms_linkEditor | MVS program name of the link editor
+bms_dependenciesDatasetMapping | DBB property mapping to map dependencies to different target datasets
```

Comment:
I do not see the `bms_dependenciesDatasetMapping` property in the PR code. Maybe unused?

NOTES:
- the rendering of this section `dependencyReport.properties` is bad (not displayed as an array)
- There is no section for REXX.properties in the README

Reply:
the dependencyReport needs to be fixed in a separate PR (there is one in DRAFT), which fixes it.